listenbrainz/webserver/static/js/src/APIService.test.ts

```diff
@@ -619,6 +621,63 @@ describe("deleteListen", () => {
   });
 });
 
+describe("getFeedForUser", () => {
+  const payload = { ...feedProps };
+  beforeEach(() => {
+    // Mock function for fetch
+    window.fetch = jest.fn().mockImplementation(() => {
+      return Promise.resolve({
+        ok: true,
+        status: 200,
+        json: () => Promise.resolve({ payload }),
+      });
+    });
+
+    // Mock function for checkStatus
+    apiService.checkStatus = jest.fn();
+  });
+
+  it("calls fetch with correct parameters", async () => {
+    await apiService.getFeedForUser("fnord", "shhh", 12345, undefined, 25);
+    expect(window.fetch).toHaveBeenCalledWith(
+      "foobar/1/user/fnord/feed/events?min_ts=12345&count=25",
+      {
+        method: "GET",
+      }
+    );
+  });
+
+  it("calls checkStatus once", async () => {
+    await apiService.getFeedForUser("fnord", "shhh");
+    expect(apiService.checkStatus).toHaveBeenCalledTimes(1);
+  });
+
+  it("returns the feed events array if successful", async () => {
+    const events = await apiService.getFeedForUser("fnord", "shhh");
+    expect(events).toBeDefined();
+    expect(events).toEqual(payload.events);
+  });
+
+  it("throws appropriate error if username is missing", async () => {
+    await expect(apiService.getFeedForUser("", "")).rejects.toThrow(
+      SyntaxError("Username missing")
+    );
+  });
+  it("throws appropriate error if userToken is missing", async () => {
+    await expect(apiService.getFeedForUser("Cthulhu", "")).rejects.toThrow(
+      SyntaxError("User token missing")
+    );
+  });
+
+  it("doesn't allow min_ts and max_ts at the same time", async () => {
```

Comment:
this is not really necessary, the endpoint is ok with getting both min_ts and max_ts